rules/code-quality/tests/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector/Fixture/property_after_instantiation.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\CodeQuality\Tests\Rector\Isset_\IssetOnPropertyObjectToPropertyExistsRector\Fixture;
+
+class SomeClass2
+{
+    public $x;
+    public $y;
+}
+
+$obj = new SomeClass2();
+isset($obj->x);
+isset($obj->y);
+isset($obj->x) && isset($obj->y);
+
```

Comment:
These snippets should be wrapped in class method call, like the one above. This file is included, so this part would get executed and cause unintended changes.

Reply:
fixed.